### ISSUE TITLE
Navigation arrows are not labeled 👉🏽

### ISSUE THREAD

**Issue by jagtalon** [duckduckgo/duckduckgo-privacy-extension#252]
Right arrow is just announced as 'link'. This should probably be a 'button' element and it should be labeled properly such as 'More details on vox.com'.

<img width="1680" alt="nav" src="https://user-images.githubusercontent.com/81969/43590087-35eb4aee-963e-11e8-9f3f-0f9c2f177855.png">
